fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -1195,6 +1195,7 @@ public FDBDatabaseRunner newRunner() {
         runner.setWeakReadSemantics(weakReadSemantics);
         runner.setPriority(priority);
         runner.setTransactionTimeoutMillis(timeoutMillis);
+        runner.setRecordLayerPropertyStorage(propertyStorage);
```

Comment:
It's minor, but we should probably update the javadoc on this method to mention that it transfers the property storage, too

Reply:
Updated the javadoc